.zenodo.json

```diff
@@ -0,0 +1,20 @@
+{
+  "creators": [
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Palomino, Jenny",
+      "orcid": "0000-0003-4879-9299"
+    },
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Wasser, Leah",
+      "orcid": "0000-0002-8177-6550"
+    }
+  ],
+  "keywords": [
```

Comment:
I'm wondering @jlpalomino  theses keywords aren't super useful.
What about
data science
earth data science
earth analytics course

Reply:
@lwasser Great point. The keywords have been updated per your suggestions. I believe this PR is ready to merge.